pytest_playwright/pytest_playwright.py

```diff
@@ -101,16 +122,31 @@ def browser_type_launch_args(pytestconfig: Any) -> Dict:
     return launch_options
 
 
-@pytest.fixture(scope="session")
+def _build_artifact_test_folder(
+    pytestconfig: Any, request: pytest.FixtureRequest, folder_or_file_name: str
+) -> str:
+    output_dir = pytestconfig.getoption("--output")
+    return os.path.join(output_dir, slugify(request.node.nodeid), folder_or_file_name)
+
+
+@pytest.fixture()
 def browser_context_args(
-    pytestconfig: Any, playwright: Playwright, device: Optional[str]
+    pytestconfig: Any,
+    playwright: Playwright,
+    device: Optional[str],
 ) -> Dict:
     context_args = {}
     if device:
         context_args.update(playwright.devices[device])
     base_url = pytestconfig.getoption("--base-url")
     if base_url:
         context_args["base_url"] = base_url
+
+    video_option = pytestconfig.getoption("--video")
```

Comment:
So, there is a command line argument? I don't see any docs changed, could you please add some documentation?

Reply:
We keep the documentation in the upstream repo. Will follow-up once its landed.
see here: https://playwright.dev/python/docs/next/test-runners